examples/resnet/resnet_main.py

```diff
@@ -0,0 +1,121 @@
+"""ResNet Train module, with some code from
+https://github.com/tensorflow/models/tree/master/resnet.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import cifar_input
+import numpy as np
+import resnet_model
+import ray
+import tensorflow as tf
+
+FLAGS = tf.app.flags.FLAGS
+tf.app.flags.DEFINE_string('train_data_path', '',
+                           'Filepattern for training data.')
+tf.app.flags.DEFINE_string('eval_data_path', '',
+                           'Filepattern for eval data')
+tf.app.flags.DEFINE_string('num_gpus', 1, 'Number of gpus to run with')
+
+@ray.remote
+def get_test(path):
+ os.environ['CUDA_VISIBLE_DEVICES'] = ''
+ with tf.device('/cpu:0'):
+  images, labels = cifar_input.build_input(path, 5000, False)
```

Comment:
We should probably do something about this because people will try to run the script on machines without GPUs.